.ci/.azure-pipelines-steps.yml

```diff
@@ -11,7 +11,7 @@ steps:
 - script: npm install
   displayName: npm install
 
-- script: npm run coveralls
+- script: npm run cover && istanbul-coveralls
```

Comment:
@damccorm With this change, couldn't they change the `npm run cover` command and have access?

Reply:
Yes 🤦‍♂️ Updated appropriately, sorry about that